lighthouse-core/test/lib/icons-test.js

```diff
@@ -301,29 +303,5 @@ describe('Icons helper', () => {
       const manifest = manifestParser(manifestSrc, EXAMPLE_MANIFEST_URL, EXAMPLE_DOC_URL);
       assert.equal(icons.pngSizedAtLeast(192, manifest.value).length, 1);
     });
-
-    it('fails with a non-png icon that has query params in url', () => {
```

Comment:
seems like both of these are still valuable, just changing the descriptions and switching the 0s to 1s (and they'll be ready to switch back and go when we do fix #789)